device-types/Cisco/ISR2951.yaml

```diff
@@ -1,10 +1,10 @@
 ---
 manufacturer: Cisco
-model: 2951 ISR
-part_number: CISCO2951/K9
-slug: cisco-2951-isr
+model: ISR2951
```

Comment:
Please leave the model as is, as that is the correct model number. This is frequently referred to as a "2951 ISR" in literature.

Reply:
Fixed